frontend/src/FindQuiz.js

```diff
@@ -48,7 +53,10 @@ class FindQuizContainer extends React.Component {
         this.rateRestaurant(true, restaurant.id);
         alert("Have a nice meal! After you eat, don't forget to rate your experience for even better recommendations!");
     }
-
+    later = (e) =>{
+        e.preventDefault()
+        //TODO: reject but snooze until later
```

Comment:
This should be pretty easy to implement if you call `rateRestaurant()` here with `rating="later"` or something. You'd also need to change the `rate_suggestion` method in app.py to check if `is_liked` is a certain string, then just cache the ID without doing anything with the user data.

Reply:
This was addressed in subsequent commits.